src/fides/api/util/consent_util.py

```diff
@@ -458,13 +478,17 @@ def prepare_privacy_notice_patches(
         check_conflicting_data_uses(
             validation_updates,
             existing_notices.values(),
-            ignore_disabled=ignore_disabled,
+            all_data_uses=DataUse.query(db).all(),
+            # ignore_disabled=ignore_disabled, # TODO: figure out if we should re-enable disabled checking of templates, how to properly allow GPP overlaps
```

Comment:
one of the major outstanding questions. i'm guessing we have good reason to _not_ ignore disabled on the template updates, so i'll need to figure out a way to let the GPP notice templates bypass this check

Reply:
ok thank you for finding that context! that's very helpful.

let me take a look and see how tricky or messy it will be to special-case the GPP notices. i suppose we may revisit in a couple of weeks anyway with the overhaul work, so it's probably not something we need to spend a ton of time figuring out?

Reply:
this ended up being pretty simple (i think), added a bypass only for disabled GPP notices in 4bfb89cbb8ee1035b3516094365d83625617c308